cfgov/templates/wagtailadmin/login.html

```diff
@@ -1,35 +1,40 @@
 {% overextends "wagtailadmin/login.html" %}
-{% load compress static i18n %}
-
+{% load static i18n %}
 
 {% block furniture %}
     <div class="content-wrapper">
-        {% if messages  or form.non_field_errors %}
+        {% if messages or form.errors %}
             <div class="messages">
                 <ul>
+                    {% if form.errors %}
```

Comment:
Is there a reason we don't want to enumerate the specific errors anymore?

Reply:
Nope, I think I was too aggressive in bringing over what I thought were Wagtail changes.